@here/harp-mapview-decoder/lib/GeoJsonTiler.ts

```diff
@@ -62,7 +62,7 @@ export class GeoJsonTiler implements ITiler {
                 buffer: 0, // tile buffer on each side
                 lineMetrics: false, // whether to calculate line metrics
                 promoteId: null, // name of a feature property to be promoted to feature.id
-                generateId: false, // whether to generate feature ids. Cannot be used with promoteId
+                generateId: true, // whether to generate feature ids. Cannot be used with promoteId
```

Comment:
Maybe this value should be taken from OmvDataSourceParameters.gatherFeatureIds, not hardcoded

Reply:
They're kind of different though, as ``generateId`` generates a unique id for every feature in the data and ``gatherFeatureIds`` fetches an already generated id (which is sometimes shared with even some other datasources).